compose/ui/ui/src/webTest/kotlin/androidx/compose/ui/input/KeyEventConversionTests.kt

```diff
@@ -0,0 +1,213 @@
+        keyDownEvent("m", code = "KeyM").assertEquivalence(key = Key.M, codePoint = 109)
+        keyDownEvent("n", code = "KeyN").assertEquivalence(key = Key.N, codePoint = 110)
+        keyDownEvent("o", code = "KeyO").assertEquivalence(key = Key.O, codePoint = 111)
+        keyDownEvent("p", code = "KeyP").assertEquivalence(key = Key.P, codePoint = 112)
+        keyDownEvent("q", code = "KeyQ").assertEquivalence(key = Key.Q, codePoint = 113)
+        keyDownEvent("r", code = "KeyR").assertEquivalence(key = Key.R, codePoint = 114)
+        keyDownEvent("s", code = "KeyS").assertEquivalence(key = Key.S, codePoint = 115)
+        keyDownEvent("t", code = "KeyT").assertEquivalence(key = Key.T, codePoint = 116)
+        keyDownEvent("u", code = "KeyU").assertEquivalence(key = Key.U, codePoint = 117)
+        keyDownEvent("v", code = "KeyV").assertEquivalence(key = Key.V, codePoint = 118)
+        keyDownEvent("w", code = "KeyW").assertEquivalence(key = Key.W, codePoint = 119)
+        keyDownEvent("x", code = "KeyX").assertEquivalence(key = Key.X, codePoint = 120)
+        keyDownEvent("y", code = "KeyY").assertEquivalence(key = Key.Y, codePoint = 121)
+        keyDownEvent("z", code = "KeyZ").assertEquivalence(key = Key.Z, codePoint = 122)
+
+        keyDownEvent("`", code = "Backquote", keyCode = Key.Grave.keyCode.toInt()).assertEquivalence(key = Key.Grave, codePoint = 96)
+        keyDownEvent("0", code = "Digit0").assertEquivalence(key = Key.Zero)
+        keyDownEvent("1", code = "Digit1").assertEquivalence(key = Key.One)
+        keyDownEvent("2", code = "Digit2").assertEquivalence(key = Key.Two)
+        keyDownEvent("3", code = "Digit3").assertEquivalence(key = Key.Three)
+        keyDownEvent("4", code = "Digit4").assertEquivalence(key = Key.Four)
+        keyDownEvent("5", code = "Digit5").assertEquivalence(key = Key.Five)
+        keyDownEvent("6", code = "Digit6").assertEquivalence(key = Key.Six)
+        keyDownEvent("7", code = "Digit7").assertEquivalence(key = Key.Seven)
+        keyDownEvent("8", code = "Digit8").assertEquivalence(key = Key.Eight)
+        keyDownEvent("9", code = "Digit9").assertEquivalence(key = Key.Nine)
+
+        keyDownEvent("0", code = "Digit0").assertEquivalence(key = Key.Zero)
+        keyDownEvent("1", code = "Digit1").assertEquivalence(key = Key.One)
+        keyDownEvent("2", code = "Digit2").assertEquivalence(key = Key.Two)
+        keyDownEvent("3", code = "Digit3").assertEquivalence(key = Key.Three)
+        keyDownEvent("4", code = "Digit4").assertEquivalence(key = Key.Four)
+        keyDownEvent("5", code = "Digit5").assertEquivalence(key = Key.Five)
+        keyDownEvent("6", code = "Digit6").assertEquivalence(key = Key.Six)
+        keyDownEvent("7", code = "Digit7").assertEquivalence(key = Key.Seven)
+        keyDownEvent("8", code = "Digit8").assertEquivalence(key = Key.Eight)
+        keyDownEvent("9", code = "Digit9").assertEquivalence(key = Key.Nine)
```

Comment:
Digit0..Digit9 are dublicated.

Reply:
Thanks for noticing - those was supposed to be section about Numpad digit keys, pushed fix